app/src/service/CategoriesService.php

```diff
@@ -0,0 +1,101 @@
+<?php
+
+namespace Src\service;
+
+use Src\db\DatabaseCon;
+
+class CategoriesService extends DatabaseCon
+{
+    private $dbConnection = null;
+
+    public function __construct()
+    {
+        $this->dbConnection = $this->getConnection();
+    }
+
+    /**
+     * to-do
+     * - find a better name for this
+     */
+    public function getAllCategories()
+    {
+        /**
+         * TO-DO
+         * - Add view counter
+         */
+        $statement = "
+            SELECT *
+            FROM categories
+            where is_deleted = 0;
+        ";
+
+        try {
+            $statement = $this->dbConnection->prepare($statement);
```

Comment:
@jjsimonAB since you are going to use a singleton  , you can wrap this on a function to avoid having to do a try catch on all the functions and have it on one

$this->connection->query(?sql);